install.sh

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 Hypermode Inc.
+# Licensed under the terms of the Apache License, Version 2.0
+# See the LICENSE file that accompanied this code for further details.
+#
+# SPDX-FileCopyrightText: 2024 Hypermode Inc. <hello@hypermode.com>
+# SPDX-License-Identifier: Apache-2.0
+
+# Run with: curl install.hypermode.com/hyp.sh -sSfL | bash
+
+# Config
```

Comment:
```suggestion
# Config
CLI_NAME="Hypermode CLI"
```

Reply:
@ryanfoxtyler should decide it it's "Hypermode CLI" or "Hyp CLI" before we merge this, but I like having it in a single variable.

Reply:
I'll update the Modus installer with that too.

Reply:
Hyp CLI

Reply:
Ryan said "Hyp CLI". 👍